src/main/java/com/amazon/opendistro/elasticsearch/performanceanalyzer/rca/framework/core/temperature/TemperatureDimension.java

```diff
@@ -0,0 +1,13 @@
+package com.amazon.opendistro.elasticsearch.performanceanalyzer.rca.framework.core.temperature;
```

Comment:
Licensing information missing from this file.

Reply:
Done